lib/utils/validation.ts

```diff
@@ -75,9 +75,13 @@ export function validateUuid(input: string): boolean {
 	return validateLongUuid(input) || validateShortUuid(input);
 }
 
-export function parseAsInteger(input: string, paramName?: string) {
+export function looksLikeInteger(input: string) {
 	// Allow only digits, no leading 0
-	if (!/^(0|[1-9][0-9]*)$/.test(input)) {
+	return !/^(0|[1-9][0-9]*)$/.test(input);
```

Comment:
The exclamation mark is probably incorrect here. It may be the reason why the tests are failing with the error:
```
[Debug]   Fetching application by id or name abc1234 (string)
Not a valid value: number

Error: Not a valid value: number
```